packages/grid/x-data-grid-pro/src/DataGridProColumnHeaders.tsx

```diff
@@ -1,8 +1,79 @@
 import * as React from 'react';
+import { unstable_composeClasses as composeClasses } from '@mui/material';
+import { styled, alpha } from '@mui/material/styles';
+import { getDataGridUtilityClass } from '../../_modules_/grid/gridClasses';
+import { useGridRootProps } from '../../_modules_/grid/hooks/utils/useGridRootProps';
+import { GridComponentProps } from '../../_modules_/grid/GridComponentProps';
 import { useGridColumnHeaders } from '../../_modules_/grid/hooks/features/columnHeaders/useGridColumnHeaders';
-import { GridScrollArea } from '../../_modules_/grid/components/GridScrollArea';
-import { GridColumnHeaders } from '../../_modules_/grid/components/columnHeaders/GridColumnHeaders';
+import { useGridApiContext } from '../../_modules_/grid/hooks/utils/useGridApiContext';
+import { useGridSelector } from '../../_modules_/grid/hooks/utils/useGridSelector';
+import { visibleGridColumnsSelector } from '../../_modules_/grid/hooks/features/columns/gridColumnsSelector';
+import { useGridApiEventHandler } from '../../_modules_/grid/hooks/utils/useGridApiEventHandler';
 import { GridColumnHeadersInner } from '../../_modules_/grid/components/columnHeaders/GridColumnHeadersInner';
+import { GridColumnHeaders } from '../../_modules_/grid/components/columnHeaders/GridColumnHeaders';
+import { gridPinnedColumnsSelector } from '../../_modules_/grid/hooks/features/columnPinning/columnPinningSelector';
```

Comment:
We should add a `_modules_/grid/hooks/features/columnPinning/index.ts` which exports `columnPinningSelector.ts` and `gridColumnPinningState.ts`

Reply:
Done.